src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -115,6 +116,22 @@ internal IEnumerable<IKeyValueAdapter> Adapters
         /// </summary>
         internal bool IsOfflineCacheConfigured { get; private set; } = false;
 
+        /// <summary>
+        /// The <see cref="ILoggerFactory"/> for creating a logger to log errors during refresh operation.
+        /// </summary>
+        internal ILoggerFactory LoggerFactory { get; set; }
+
+        /// <summary>
+        /// Register a logger factory to log errors during refresh operations.
```

Comment:
I would opt for generalizing the description a bit more. We may log other things in the future, like startup failures.